StockAnalysis.Tests/DiffTests/DiffStoreTests/DiffStoreCreatorTests.cs

```diff
@@ -0,0 +1,97 @@
+using StockAnalysis.Diff.Store;
+
+namespace StockAnalysisTests.DiffTests.DiffStoreTests;
+
+public class DiffStoreCreatorTests
+{
+    [Test]
+    public void CreateStore_Csv_RegularFormat_Succeeds()
+    {
+        try
```

Comment:
Try blok se v tomhle pripade psat nemusi. pokud tam je neosetrena vyjimka, test failne, Nunit to poresi a vypise obvykle neco smysluplneho.